dotcom-rendering/src/layouts/FullPageInteractiveLayout.tsx

```diff
@@ -180,39 +196,66 @@ const NavHeader = ({ article, NAV, format }: Props) => {
 	);
 };
 
-export const FullPageInteractiveLayout = ({ article, NAV, format }: Props) => {
+export const FullPageInteractiveLayout = (props: WebProps | AppsProps) => {
+	const { article, format, renderingTarget } = props;
 	const {
 		config: { host },
 		editionId,
 	} = article;
+	const isWeb = renderingTarget === 'Web';
+	const isApps = renderingTarget === 'Apps';
 
 	return (
 		<>
 			{article.isLegacyInteractive && (
 				<Global styles={interactiveGlobalStyles} />
 			)}
-			<header
-				css={css`
-					background-color: ${themePalette('--article-background')};
-				`}
-			>
-				<NavHeader article={article} NAV={NAV} format={format} />
+			{isWeb && (
+				<>
+					<header
+						css={css`
+							background-color: ${themePalette(
+								'--article-background',
+							)};
+						`}
+					>
+						<NavHeader
+							article={article}
+							NAV={props.NAV}
+							format={format}
+						/>
 
-				{format.theme === ArticleSpecial.Labs && (
-					<Stuck>
-						<Section
-							fullWidth={true}
-							showTopBorder={false}
-							padSides={true}
-							backgroundColour={sourcePalette.labs[400]}
-							borderColour={sourcePalette.neutral[60]}
-							sectionId="labs-header"
-						>
-							<LabsHeader editionId={editionId} />
-						</Section>
-					</Stuck>
-				)}
-			</header>
+						{format.theme === ArticleSpecial.Labs && (
+							<Stuck>
+								<Section
+									fullWidth={true}
+									showTopBorder={false}
+									padSides={true}
+									backgroundColour={sourcePalette.labs[400]}
+									borderColour={sourcePalette.neutral[60]}
+									sectionId="labs-header"
+								>
+									<LabsHeader editionId={editionId} />
+								</Section>
+							</Stuck>
+						)}
+					</header>
+				</>
+			)}
+			{isApps && format.theme === ArticleSpecial.Labs && (
+				<Stuck>
+					<Section
+						fullWidth={true}
+						showTopBorder={false}
+						padSides={true}
+						backgroundColour={sourcePalette.labs[400]}
+						borderColour={sourcePalette.neutral[60]}
+						sectionId="labs-header"
+					>
+						<LabsHeader editionId={editionId} />
+					</Section>
+				</Stuck>
```

Comment:
You may want to keep this in a `header` for semantic consistency?

```suggestion
				<header>
					<Stuck>
						<Section
							fullWidth={true}
							showTopBorder={false}
							padSides={true}
							backgroundColour={sourcePalette.labs[400]}
							borderColour={sourcePalette.neutral[60]}
							sectionId="labs-header"
						>
							<LabsHeader editionId={editionId} />
						</Section>
					</Stuck>
				</header>
```